src/parsers/shift.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import pkg from 'shift-parser/package.json';
+import loadAndExecute from './utils/loadAndExecute';
```

Comment:
Just spelling not a big deal https://github.com/fkling/esprima_ast_explorer/blob/cbf1b2da0339c6e4af317a8ae462731e840346d9/src/parsers/acorn.js#L3